src/content/app/genome-browser/components/track-panel/components/track-panel-tabs/TrackPanelTabs.tsx

```diff
@@ -77,21 +97,27 @@ export const TrackPanelTabs = () => {
 
     return classNames(styles.trackPanelTab, {
       [styles.trackPanelTabActive]: isTrackPanelTabActive,
-      [styles.trackPanelTabDisabled]: !focusObject?.genome_id
+      [styles.trackPanelTabDisabled]: isDisabled
     });
   };
 
   return (
     <div className={`${styles.trackPanelTabs}`}>
-      {Object.values(TrackSet).map((value: TrackSet) => (
-        <span
-          className={getTrackPanelTabClassNames(value)}
-          key={value}
-          onClick={() => handleTabClick(value)}
-        >
-          {value}
-        </span>
-      ))}
+      {Object.values(TrackSet).map((tabName: TrackSet) => {
+        const isTabDisabled =
+          !focusObject?.genome_id || !hasTracksInCategory(tabName);
```

Comment:
In which kind of scenario is the `genome_id` not going to be available?

Reply:
If `getBrowserActiveFocusObject` returns `null`, which it technically [is allowed to do](https://github.com/Ensembl/ensembl-client/blob/d8c8c7715c6e123b6be147c8085c8e64e228820a/src/content/app/genome-browser/state/browser-general/browserGeneralSelectors.ts#L41-L47).